src/app/main-layout/main-layout.module.ts

```diff
@@ -11,24 +11,28 @@ import { HeaderComponent } from '../header/header.component';
 import { AsideComponent } from '../aside/aside.component';
 import { FooterComponent } from '../footer/footer.component';
 import { NotFoundComponent } from '../not-found/not-found.component';
+import { MySkillsModule } from '../my-skills/my-skills.module';
+import { CoursesModule } from '../courses/courses.module';
+import { ExamplesModule } from '../examples/examples.module';
+import { ContactsModule } from '../contacts/contacts.module';
 
 
 @NgModule({
   declarations: [
     MainLayoutComponent,
     HomeComponent,
-    CoursesComponent,
-    MySkillsComponent,
-    ExamplesComponent,
-    ContactsComponent,
     HeaderComponent,
     AsideComponent,
     FooterComponent,
     NotFoundComponent
   ],
   imports: [
     CommonModule,
-    MainLayoutRoutingModule
+    MainLayoutRoutingModule,
```

Comment:
@Yuriy-27  you should not import those modules which are loaded as lazy.
They will be loaded in runtime in the browser when they will be needed due to route url